### ISSUE TITLE
Server events in Disc

### ISSUE THREAD

**Issue by ForTheWout** [ForTheCrown/FTC#26]
It'd be cool if the FTC Bot displayed some events in one of the discord channels, such as:
- [ ] End closing and opening
- [ ] Global guild exp activated (weekend one too?)
- [ ] Resource world reset
- [ ] Player has completed `/shop` challenge 10, 20, 30, 50 times in a row
- [ ] Player has broken `/shop` challenge all time record
- [ ] ...

**Comment by ForTheWout** [comment]
Yeah we can send stuff that's relevant for all players to a general channel. 
Maybe guild related events can be displayed in a guild channel? Like a new player joining, or when someone unlocks something?

**Comment by JulieWoolie** [comment]
What channel should this be broadcast in? End closing and RW reset could be `#updates`, no?

**Comment by ForTheWout** [comment]
Yeah sure, or you can post them in `#chat` that's prob fine too
